graph_queries/README.md

```diff
@@ -0,0 +1,33 @@
+# Graph queries
+
+## Introduction
+
+This directory has a list of sample graph queries that you can use in IBA probes. The
+goal is to help probe authors get familiar with graph queries in the process of
+creating probes.
+
+Refer to AOS documentation on [graph queries](
+https://files.apstra.com/docs/database_concepts.html#query-specification) for more
+information about the query syntax.
+
+Each example is presented in a YAML file, simply for better organization and
+readability. These YAML documents are __not__ consumed by AOS or any other software
+component.
+
+## Disclaimer
+
+Exact graph query to use depends a lot on the specific use case in question. The
+queries stated here are only for illustration and not meant to be used "as is".
+
+Each query has a lot of choices to make and it's very important for you to understand
+your exact requirement and come up with the right graph query(ies) to be used in
+probes. You can certainly use these samples as starting points but it's unlikely for
+any of these to be an accurate match for your specific requirements.
+
+## Interactive query execution
+
+Apstra Operating System GUI has 'GraphQL API Explorer' accessible from the blueprint
+details view. You can execute the queries in this directory in the explorer
+and examine the results. You can also tweak the query and see the results
+interactively. Use the explorer as an aid to arrive at the final queries that work
+for your specific requirements.
```

Comment:
Might be good to specify that these example are QE queries.

Reply:
That thought did cross my mind but I felt given this is an IBA repo, the queries here work for IBA and the readers don't need to be worried about other query languages we may support. Wdyt? I'll mention it in some form I guess.

Reply:
I meant that if they paste the query that's shown into GraphQL API Explorer they won't work unless they select qe.